frontend/pages/product_new_page.js

```diff
@@ -3,6 +3,7 @@ import FormInput from 'src/frontend/components/form_input';
 import http from 'src/frontend/services/http';
 import { Redirect } from 'react-router';
 import appRoutes from 'src/constants/routes';
+import { identifier } from 'babel-types';
```

Comment:
what's this for?

Reply:
Not me

Reply:
Wait. Idk. I didn't add it... Idk how it got added